Gems/Atom/Bootstrap/Code/Source/BootstrapSystemComponent.cpp

```diff
@@ -681,6 +666,36 @@ namespace AZ
                 return true;
             }
 
+            void BootstrapSystemComponent::RunBRDFPipeline(AZ::RPI::ScenePtr scene, AZ::RPI::ViewportContextPtr viewportContext)
+            {
+               // As part of our initialization we need to create the BRDF texture generation pipeline
+               AZ::RPI::RenderPipelineDescriptor pipelineDesc;
+               pipelineDesc.m_mainViewTagName = "MainCamera";
+               pipelineDesc.m_name = AZStd::string::format("BRDFTexturePipeline_%i", viewportContext->GetId());
```

Comment:
Isn't it possible that viewportContext is null here in the headless renderer?

We might want to bail out real early if it is.

Reply:
@nick-l-o3de I've modified cherry-picked version to use code from development and crated PR https://github.com/RobotecAI/o3de/pull/12.